packages/react-form/README.md

```diff
@@ -659,10 +659,17 @@ const emptyCardFactory = (): Card => ({
   cvv: '',
 });
 
-const {fields, addItem, removeItem, moveItem, reset, dirty} = useDynamicList(
-  [],
-  emptyCardFactory,
-);
+const {
+  fields,
+  addItem,
+  removeItem,
+  moveItem,
+  reset,
+  dirty,
+  value,
+  newDefaultValue,
```

Comment:
This updates what the new default value is. Having `newDefaultValue` and `DefaultValue` side by side seems odd. Could we rename to `updateDefaultValue` as that is what it is doing?

Reply:
I sort of wanted to follow the convention of `Field` where the function is called `newDefaultValue` and it also has `defaultValue`. This way its consistent and easily recognizable by those that use `Field`. Wdyt?